packages/cells/src/widget.ts

```diff
@@ -1564,7 +1578,10 @@ export class MarkdownCell extends AttachmentsCell<IMarkdownCellModel> {
     if (value === this._rendered) {
       return;
     }
+
     this._rendered = value;
```

Comment:
I suggest removing the private attribute `_rendered` to have a single source of truth in the model (that will avoid troubles of duplicated information).